### DISCUSSION TITLE
Allow Unrelated Histories error while pulling push-dir deploy from github

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#14555] [General - Components / Directives / etc]
Hi everyone.. I use push-dir package to deploy builded app into github repo. There is no problem on deploying but when I pull that repo into my production server I get "refusing to merge unrelated histories" error.. this error related with index.html I think.. because when I run pull commit with  --allow-unrelated-histories flag, git gives merge error on index.html. Is there anyone else who use push-dir and getting this error? Thank you.

**Comment by MilosPaunovic** [comment]
This doesn't sound like a Quasar related issue. Best suggestion would be to ask this question on the repository of `push-dir` package.

**Reply by mecjos** [reply]
Yes it's not quasar issue but I thought if there is a quasar user who has same issue, can help. I opened an issue in push-dir repo but no answer..
